ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/lldp.j2

```diff
@@ -0,0 +1,20 @@
+{# eos - lldp #}
+{% if lldp is defined and lldp is not none %}
+{%   if lldp.timer is defined and  lldp.timer is not none %}
+lldp timer {{  lldp.timer }}
+{%   endif %}
+{%   if lldp.holdtime is defined and  lldp.holdtime is not none %}
+lldp hold-time {{  lldp.holdtime }}
+{%   endif %}
+{%   if lldp.management_address is defined and  lldp.management_address is not none %}
+lldp management-address {{  lldp.management_address }}
+{%   endif %}
+{%   if lldp.vrf is defined and  lldp.vrf is not none %}
+lldp management-address vrf {{  lldp.vrf }}
+{%   endif %}
+lldp run
```

Comment:
lldp.run should be a key that can be turned on or off regardless of the rest of the lldp configuration.
```
lldp:
  run: < true|false >
```

Reply:
Updated according to requested change

Reply:
Updated to requested change